packages/mdl-typography/README.md

```diff
@@ -0,0 +1,134 @@
+# mdl-typography
+
+MDL typography is a CSS-only component that implements the
+[Material Design typography guidelines](https://material.google.com/style/typography.html), and makes them available to
+developers as CSS classes.
+
+## Installation
+
+> Note: Installation via the npm registry will be available after alpha.
+
+
+## Usage
```

Comment:
That's where a good group of examples for us will do nicely. Our usage has traditionally been a step-by-step instructions on how to use it. We should probably kill "Examples" and move that content into "Usage".